### ISSUE TITLE
Error: Unexpected key 'disableSourceOverride' found in params

### ISSUE THREAD

**Issue by marktarry-mobysoft** [aws-actions/aws-codebuild-run-build#122]
![image](https://user-images.githubusercontent.com/68902824/217885382-0d2f1928-8884-417c-a9c8-5fac2a2672d1.png)

This error started to appear today - shortly after the [v1.0.11](https://github.com/aws-actions/aws-codebuild-run-build/releases/tag/v1.0.11) release was published.

Our (reusable) workflow file:
```yaml
on:
  workflow_call:
    inputs:
      codebuild-project-name:
        description: The name of the AWS CodeBuild project to be executed
        type: string
        required: true

jobs:
  post-deployment-testing:
    runs-on: ubuntu-latest
    steps:
      - uses: aws-actions/configure-aws-credentials@v1
        with:
          # Omitted
      - uses: aws-actions/aws-codebuild-run-build@v1
        with:
          project-name: ${{ inputs.codebuild-project-name }}
```

The workflow run logs confirm the exact action version used:
```shell
Download action repository 'aws-actions/configure-aws-credentials@v1' (SHA:67fbcbb121271f7775d2e7715933280b06314838)
Download action repository 'aws-actions/aws-codebuild-run-build@v1' (SHA:5c0236844bac232ef4c8ae94697cf7ddead5655a)
```

Note: 5c0236844bac232ef4c8ae94697cf7ddead5655a matches release v1.0.11

**Comment by taoyong-ty** [comment]
v1 release has already been rolled back, you should see recovery now

I am looking into the a fix right now

**Comment by taoyong-ty** [comment]
Thanks for reporting this error. 

The issue in v1.0.11 has been fixed by https://github.com/aws-actions/aws-codebuild-run-build/pull/124 I just re-ran the release flow to push this fix to v1 and v1.0.11 again (all pointing to ce13c105cf801565a0cfa7e461d95c6a2a2ef16b). I've also verified the end-to-end functionality of the new version

Please re-open if you still encounter any issue.
